src/index.jsx

```diff
@@ -49,6 +49,8 @@ subscribe(APP_READY, () => {
     'edx.bi.ecommerce.payment_mfe.started_painting',
     getPerformanceProperties(),
   );
+  sendRev1074Event('payment_mfe.started_painting', {});
```

Comment:
you're adding all of these after sendTrackEvent but doesn't that function go to the same api?

Reply:
No, I believe the regular track events are going to Segment.

Reply:
Mainly they're going to different destinations; by adding "duplicate" events going to the API we plan to use for the Static Checkout Page test we can validate that mechanism and try out analyzing these events.